src/static/pricing-page/index.html

```diff
@@ -648,10 +638,7 @@ <h1>Frequently Asked Questions</h1>
             </div>
 
             <div id="collapseFive" class="PricePage__faq__panel-group__body panel-collapse collapse" role="tabpanel" aria-labelledby="headingFive">
-                Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt
-                laborum eiusmod. Brunch 3 wolf moon tempor, sunt aliqua put a bird on it squid single-origin coffee nulla assumenda shoreditch et. Nihil anim keffiyeh helvetica, craft beer labore wes anderson
-                cred nesciunt sapiente ea proident. Ad vegan excepteur butcher vice lomo.
-                Leggings occaecat craft beer farm-to-table, raw denim aesthetic synth nesciunt you probably haven't heard of them accusamus labore sustainable VHS.
+                Every Collective that receives or pays out money needs a fiscal host, because they need to connect the host's bank account, Stripe account, and Paypal account to activate Open Collective’s payment features. If you already have a legal company set up, or you are comfortable taking on the responsibility as an individual, you can [fiscally host your own Collective](https://docs.opencollective.com/help/hosts/become-host).
```

Comment:
```suggestion
                Every Collective that receives or pays out money needs a fiscal host because they need to connect the host's bank account, Stripe account, and Paypal account to activate Open Collective’s payment features. If you already have a legal company set up, or you are comfortable taking on the responsibility as an individual, you can [fiscally host your own Collective](https://docs.opencollective.com/help/hosts/become-host).
```